wger/nutrition/management/commands/import-off-products.py

```diff
@@ -70,15 +71,63 @@ def add_arguments(self, parser):
             'completeness score that ranges from 0 to 1.1. Default: 0.7'
         )
 
-    def handle(self, **options):
-
+        parser.add_argument(
+            '--jsonl',
+            action='store_true',
+            default=False,
+            dest='usejsonl',
+            help='Use the JSONL dump of the Open Food Facts database.'
+            '(this option does not require mongo)'
+        )
+    
+    def products_mongo(self,filterdict):
+        """returns a mongo iterator with filtered prodcuts"""
         try:
             # Third Party
             from pymongo import MongoClient
         except ImportError:
             self.stdout.write('Please install pymongo, `pip install pymongo`')
             return
 
+        client = MongoClient('mongodb://off:off-wger@127.0.0.1', port=27017)
+        db = client.admin
+        return db.products.find(filterdict)
+    
+    def products_jsonl(self,languages,completeness):
+        import json
+        import requests
+        from gzip import GzipFile
+        off_url='https://static.openfoodfacts.org/data/openfoodfacts-products.jsonl.gz'
```

Comment:
should this be a default in env file? That way if the URL changes, people can update with a hotfix to env file instead of pulling in a new release?

Reply:
I was thinking that only "upstream" wger regularly does this and other instances sync from us like with the exercises. Then we don't generate so much traffic for OFF (and don't need to respond as fast to such changes, if they ever occur)